addons/bestja_project_hierarchy/models.py

```diff
@@ -155,6 +155,14 @@ class Project(models.Model):
         related='organization.level'
     )
 
+    _sql_constraints = [
+        (
+            'invitation_parent_project_organization_uniq',
+            'unique(parent, organization)',
+            "Organizacja może zostać zaproszona do danego projektu przez organizację nadrzedną tylko raz!"
```

Comment:
This constraint have nothing to do with sending invitations, I think the message should read "Nie możesz utworzyć wielu projektu na podstawie jednego projektu nadrzędnego" or something like that.